### ISSUE TITLE
Refactor Examples

### ISSUE THREAD

**Issue by ac74475** [gchq/Palisade#222]
The Examples are starting to get messy and unwieldy.

I would like to simplify things down to having the following examples:
- local machine JVM example
- local machine docker example
- AWS EMR deployment example (issue gh-114 is adding this capability)
- Azure HDInsights example (issue gh-162 is adding this capability)

The local machine docker example, would be very similar to the current docker example. I would like to see the multi-jvm-example-... modules be under a 'example-services' module and remove the multi-jvm-example- prefix so they are rest-X-service. Then there should be a deployment directory in the example module which has a sub-directory 'bash' with a subdirectory called 'local docker', which contains the docker specific files and scripts.

THEN update all the Readme.MD files!

The AWS EMR deployment should be a bunch of scripts in example/deployment/bash/aws_emr to deploy Palisade on an AWS EMR cluster by: 
- starting up a etcd cluster over all the data nodes
- starting up a rest config service on the master node (using the example-rest-config-service executable jar)
- running a services configurator which sets all the configuration for the services to be deployed
- starting up the remaining palisade services (data services running on each of the data nodes and the other services just running on the master)
- running an example model configurator which sets up the users and policies
- running the example runner that runs the example queries to demo the capability

The Azure HDInsight deployment should be very similar to the AWS EMR one.

**Comment by m09526** [comment]
Blocked by gh-114

**Comment by developer6959** [comment]
Ready for PR and review

**Comment by ac74475** [comment]
merged into develop
